diff -rupN gzdoom-g2.1.1.orig/src/CMakeLists.txt gzdoom-g2.1.1/src/CMakeLists.txt
--- gzdoom-g2.1.1.orig/src/CMakeLists.txt	2016-02-23 01:20:43.000000000 -0800
+++ gzdoom-g2.1.1/src/CMakeLists.txt	2016-09-08 00:02:34.906037527 -0700
@@ -50,7 +50,8 @@ endif( CMAKE_SIZEOF_VOID_P MATCHES "8" )
 # a system-wide version.
 
 # Construct version numbers for searching for the FMOD library on Linux.
-set( MINOR_VERSIONS "50" "49" "48" "47" "46" "45" "44" "43" "42" "41"
+set( MINOR_VERSIONS "63" "62" "61" "60" "59" "58" "57" "56" "55" "54"
+	"53" "52" "51" "50" "49" "48" "47" "46" "45" "44" "43" "42" "41"
 	"40" "39" "38" "37" "36" "35" "34" "33" "32" "31" "30" "29" "28"
 	"27" "26" "25" "24" "23" "22" "21" "20" "21" "19" "18" "17" "16"
 	"15" "14" "13" "12" "11" "10" "09" "08" "07" "06" "05" "04" "03"
