diff -rupN nginx-1.4.4.orig/conf/nginx.conf nginx-1.4.4/conf/nginx.conf
--- nginx-1.4.4.orig/conf/nginx.conf	2013-11-19 03:25:24.000000000 -0800
+++ nginx-1.4.4/conf/nginx.conf	2014-01-17 16:31:15.543135999 -0800
@@ -41,7 +41,7 @@ http {
         #access_log  logs/host.access.log  main;
 
         location / {
-            root   html;
+            root   /var/www/ngdocs;
             index  index.html index.htm;
         }
 
@@ -51,7 +51,7 @@ http {
         #
         error_page   500 502 503 504  /50x.html;
         location = /50x.html {
-            root   html;
+            root   /var/www/ngdocs;
         }
 
         # proxy the PHP scripts to Apache listening on 127.0.0.1:80
@@ -63,10 +63,10 @@ http {
         # pass the PHP scripts to FastCGI server listening on 127.0.0.1:9000
         #
         #location ~ \.php$ {
-        #    root           html;
+        #    root           /var/www/ngdocs;
         #    fastcgi_pass   127.0.0.1:9000;
         #    fastcgi_index  index.php;
-        #    fastcgi_param  SCRIPT_FILENAME  /scripts$fastcgi_script_name;
+        #    fastcgi_param  SCRIPT_FILENAME  $document_root$fastcgi_script_name;
         #    include        fastcgi_params;
         #}
 
@@ -87,7 +87,7 @@ http {
     #    server_name  somename  alias  another.alias;
 
     #    location / {
-    #        root   html;
+    #        root   /var/www/ngdocs;
     #        index  index.html index.htm;
     #    }
     #}
@@ -110,7 +110,7 @@ http {
     #    ssl_prefer_server_ciphers   on;
 
     #    location / {
-    #        root   html;
+    #        root   /var/www/ngdocs;
     #        index  index.html index.htm;
     #    }
     #}
